docs/remote_inference_blueprints/bedrock_connector_rerank_blueprint.md

```diff
@@ -0,0 +1,235 @@
+            "type": "BEDROCK_RERANKING_MODEL"
+          },
+          "sources": ${parameters.sources}
+        }
+      """,
+      "post_process_function": "connector.post_process.bedrock.rerank"
+    }
+  ]
+}
+```
+
+If you are using Amazon OpenSearch Service, you can provide an AWS Identity and Access Management (IAM) role Amazon Resource Name (ARN) that allows access to Amazon Bedrock. For more information, see the [AWS documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ml-amazon-connector.html). Use the following request to create a connector:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Amazon Bedrock Rerank API",
+  "description": "Test connector for Amazon Bedrock Rerank API",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+      "roleArn": "<PLEASE ADD YOUR AWS ROLE ARN HERE>"
+  },
+  "parameters": {
+    "service_name": "bedrock",
+    "endpoint": "bedrock-agent-runtime",
+    "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+    "api_name": "rerank",
+    "model_id": "amazon.rerank-v1:0"
+  },
+  "actions": [
+    {
+      "action_type": "PREDICT",
+      "method": "POST",
+      "url": "https://${parameters.endpoint}.${parameters.region}.amazonaws.com/${parameters.api_name}",
+      "headers": {
+        "x-amz-content-sha256": "required",
+        "content-type": "application/json"
+      },
+      "pre_process_function": "connector.pre_process.bedrock.rerank",
+      "request_body": """
+        {
+          "queries": ${parameters.queries},
+          "rerankingConfiguration": {
+            "bedrockRerankingConfiguration": {
+              "modelConfiguration": {
+                "modelArn": "arn:aws:bedrock:${parameters.region}::foundation-model/${parameters.model_id}"
+              }
+            },
+            "type": "BEDROCK_RERANKING_MODEL"
+          },
+          "sources": ${parameters.sources}
+        }
+      """,
+      "post_process_function": "connector.post_process.bedrock.rerank"
+    }
+  ]
+}
+```
+
+Sample response:
+```json
+{
+  "connector_id": "nMopmY8B8aiZvtEZLu9B"
+}
+```
+
+## 3. Create model group:
+
+```json
+POST /_plugins/_ml/model_groups/_register
+{
+  "name": "remote_model_group_rerank",
+  "description": "This is an example description"
+}
+```
+
+Sample response:
+```json
+{
+  "model_group_id": "IMobmY8B8aiZvtEZeO_i",
+  "status": "CREATED"
+}
+```
+
+## 4. Register model to model group & deploy model:
+
+```json
+POST /_plugins/_ml/models/_register?deploy=true
+{
+  "name": "Amazon Bedrock Rerank API",
+  "function_name": "remote",
+  "model_group_id": "IMobmY8B8aiZvtEZeO_i",
+  "description": "test Amazon Bedrock Rerank API",
+  "connector_id": "nMopmY8B8aiZvtEZLu9B"
+}
+```
+
+Sample response:
+```json
+{
+  "task_id": "rMormY8B8aiZvtEZIO_j",
+  "status": "CREATED",
+  "model_id": "rcormY8B8aiZvtEZIe89"
+}
+```
+
+## 5. Test model inference
+
+```json
+POST _plugins/_ml/_predict/text_similarity/rcormY8B8aiZvtEZIe89
```

Comment:
See such error when use `us-east-1` to create connector: 
```
"Error from remote service: {\"message\":\"You are not authorized to invoke the Rerank operation.\"}"
```
Checked Bedrock model access and see this rerank model not ready on us-east-1, change to us-west-2 can work .

Reply:
Sorry that missed this comment. This looks good. Approved this PR. Thanks a lot for the contribution.